library/std/src/Std/BlochSphere.qs

```diff
@@ -0,0 +1,95 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/// # Summary
+/// Maps a Pauli axis from one direction to another by applying an appropriate Clifford transformation.
+/// For example, use `within MapPauliAxis(PauliZ, PauliX, q)` and perform Z rotation when X rotation is desired.
+///
+/// # Description
+/// This function applies single-qubit Clifford transformations that remap Pauli axes on a Bloch sphere.
+/// These mappings are useful to apply operations known for one Pauli basis in different Pauli bases.
+/// Provide `from` and `to` parameters in terms of a passive transformation. For example,
+/// when a rotation around the X axis is desired and a rotation around the Z axis is available,
+/// the Z axis gets pointed in the direction of the X axis so that the rotation around Z axis can be used.
+/// Out of several transformations that achieve the requested mapping, the one with fewer gates is used.
+///
+/// # Remarks
+/// The complete list of possible mappings and a list of gate sequences to achieve them.
+/// For example, a transformation of +X+Y+Z ↦ +X+Z-Y means that the bloch sphere is rotated so that
+/// the X axis remains unchanged, Y axis points in Z direction, and Z axis points in -Y direction.
+/// (Y with direction reversed). Such transformation could be used to achieve PauliZ to PauliY mapping.
+///
+/// +X+Y+Z ↦ +X+Y+Z: I (used when `from` == `to`)            \
+/// +X+Y+Z ↦ +Z-Y+X: H (used when mapping Z to X and X to Z) \
+/// +X+Y+Z ↦ +Y+Z+X: S⁻¹H (used when mapping Z to Y)         \
+/// +X+Y+Z ↦ +Z+X+Y: HS (used when mapping Y to Z)           \
+/// +X+Y+Z ↦ +Y-X+Z: S (used when mapping Y to X)            \
+/// +X+Y+Z ↦ -Y+X+Z: S⁻¹ (used when mapping X to Y)          \
+/// +X+Y+Z ↦ -X+Z+Y: S⁻¹HS                                   \
+/// +X+Y+Z ↦ -Y-Z+X: SH                                      \
+/// +X+Y+Z ↦ +Y-Z-X: SHZ, S⁻¹HY, YSH, SXH, S⁻¹YH, XS⁻¹H      \
+/// +X+Y+Z ↦ -X-Z-Y: SHS⁻¹                                   \
+/// +X+Y+Z ↦ +X-Y-Z: X                                       \
+/// +X+Y+Z ↦ -Z+X-Y: HXS⁻¹, HSX, ZHS⁻¹, YHS, HYS, HS⁻¹Y      \
+/// +X+Y+Z ↦ +X-Z+Y: SHS, HS⁻¹H                              \
+/// +X+Y+Z ↦ -Y+Z-X: SHY, S⁻¹HZ, YS⁻¹H, SYH, S⁻¹XH, XSH      \
+/// +X+Y+Z ↦ -X+Y-Z: Y                                       \
+/// +X+Y+Z ↦ -Z+Y+X: HX, ZH                                  \
+/// +X+Y+Z ↦ -Y-X-Z: YS, SX, S⁻¹Y, XS⁻¹                      \
+/// +X+Y+Z ↦ +Y+X-Z: YS⁻¹, SY, S⁻¹X, XS                      \
+/// +X+Y+Z ↦ +Z+Y-X: XH, HZ                                  \
+/// +X+Y+Z ↦ +X+Z-Y: S⁻¹HS⁻¹, HSH                            \
+/// +X+Y+Z ↦ -Z-X+Y: HXS, HSY, ZHS, YHS⁻¹, HYS⁻¹, HS⁻¹X      \
```

Comment:
I think it's worth double checking that these unicode characters show up as expected in the VS Code renering of stdlib docs.

Reply:
In my VSCode they show correctly. I did try this before using them. But the concern is valid. However, we already have unusual Unicode characters in other doc comments...